WNPRC_Purchasing/src/client/RequestEntry/RequestEntry.tsx

```diff
@@ -100,14 +118,14 @@ export const App: FC = memo(() => {
 
                         setPurchaseAdminModel(
                             PurchaseAdminModel.create({
-                                assignedTo: requestOrderVals[0].assignedTo,
-                                paymentOption: requestOrderVals[0].paymentOptionId,
-                                qcState: requestOrderVals[0].qcState,
-                                program: requestOrderVals[0].program,
-                                confirmationNum: requestOrderVals[0].confirmationNum,
-                                invoiceNum: requestOrderVals[0].invoiceNum,
-                                orderDate: requestOrderVals[0].orderDate ? new Date(requestOrderVals[0].orderDate) : null,
-                                cardPostDate: requestOrderVals[0].cardPostDate ? new Date(requestOrderVals[0].cardPostDate) : null,
+                                assignedTo: isReorder ? "" : requestOrderVals[0].assignedTo,
+                                paymentOption: isReorder ? "" : requestOrderVals[0].paymentOptionId,
+                                qcState: isReorder ? pendingState : requestOrderVals[0].qcState,
+                                program: isReorder ? "4" : requestOrderVals[0].program,
```

Comment:
Can this be a constant? I assume that we have a similar default used when we create brand new requests?